esp32/modesp.c

```diff
@@ -37,6 +38,47 @@
 #include "drivers/dht/dht.h"
 #include "modesp.h"
 
+#define MODESP_INCLUDE_CONSTANTS (1)
+
+STATIC mp_obj_t esp_osdebug(size_t n_args, const mp_obj_t *args) {
+    switch (n_args) {
+    case 1:
+        if (args[0] == mp_const_none) {
+            esp_log_level_set("*", ESP_LOG_ERROR);
+        } else {
+            esp_log_level_set("*", LOG_LOCAL_LEVEL);
+        }
+        break;
+
+    case 2:
+        esp_log_level_set("*", mp_obj_get_int(args[1]));
+        break;
+    }
+
+    return mp_const_none;
+}
+STATIC MP_DEFINE_CONST_FUN_OBJ_VAR_BETWEEN(esp_osdebug_obj, 1, 2, esp_osdebug);
```

Comment:
Maybe? You're the maintainer, so get the final say. 

The reasoning for the additional parameter is twofold: that repurposing the existing argument would break backwards compatibility for anyone migrating code from ESP8266 in a minor but unpredictable way; the other is that although I can't see how to do it yet, the ESP32 does have two UARTs, and the hardware allows developers to change pin assignments for how UARTs are routed... it may be that at some point this would become possible.

But would appreciate your thoughts.